### PR TITLE
Added a navigation.

### PR DESCRIPTION
**Pull request by islamnurdin**
Created links to specific sources. Let me know if something else is needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ### What is this repo for
 This repo is created for saving resources we found useful by to date. Mostly contains **React** and **Web development** resources.
```
Comment (ofarukcaki) [inline README.md:5]: Need header

File: README.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
 
+
+[Tools](#tools) [Authentication](#authentication) [CSS](#css) ...
+
```
Comment (ofarukcaki) [inline README.md:8]: Need to ocntains all titles

**Review by ofarukcaki — CHANGES_REQUESTED** [review]
Should contain all titles instead of few

**Comment by islamnurdin** [comment]
Give me a few minutes.

File: README.md (hunk 1 above)
Comment (islamnurdin) [reply README.md:5]: What kind of header? and What text should it contain?

**Comment by islamnurdin** [comment]
Please see the new changes.

**Pull request closed without merging**